charts/mlrun-ce/values.yaml

```diff
@@ -405,3 +405,63 @@ kube-prometheus-stack:
     fullnameOverride: state-metrics
   prometheus-node-exporter:
     fullnameOverride: node-exporter
+
+tdengine:
+  enabled: true
+  fullnameOverride: tdengine-tsdb
+
+  # For more detailes
+  # https://github.com/taosdata/TDengine-Operator/blob/3.0/helm/tdengine/values.yaml
+  replicaCount: 1
+
+  image:
+    prefix: tdengine/tdengine
+    pullPolicy: Always
+    tag: "3.0.2.2"
+
+  service:
+    type: ClusterIP
+    ports:
+      tcp: [ 6030, 6041, 6042, 6043, 6044, 6046, 6047, 6048, 6049, 6060 ]
+      udp: [ 6044, 6045 ]
+
+  # Set timezone here, not in taoscfg
+  timezone: "Europe/London"
```

Comment:
Is there not an option for UTC?
London time / GMT shift between UTC+0 and UTC+1 during summer time. To keep consistency between all our products - UTC is a much better option.

Reply:
Done